vm/src/obj/objfloat.rs

```diff
@@ -100,7 +100,7 @@ fn float_lt(vm: &mut VirtualMachine, args: PyFuncArgs) -> PyResult {
             .ctx
             .new_bool(v1 < objint::get_value(i2).to_f64().unwrap()))
     } else {
-        Err(vm.new_type_error(format!("Cannot compare {} and {}", i.borrow(), i2.borrow())))
+        Ok(vm.ctx.not_implemented())
```

Comment:
I think `float_{gt,ge,le}` need this change as well

Reply:
Done. Thanks